includes/managers/controls.php

```diff
@@ -1325,6 +1325,21 @@ public function add_motion_effects_promotion_control( Controls_Stack $controls_s
 		);
 	}
 
+	public function add_header_promotion_control( Controls_Stack $controls_stack ) {
+		if ( Utils::has_pro() ) {
+			return;
+		}
+
+		$controls_stack->add_control(
+			'animated_heading_promotion',
+			[
+				'type'      => self::RAW_HTML,
+				'separator' => 'before',
+				'raw'       => $this->react_promotion_control( esc_html__( 'Animated Widget heading', 'elementor' ), 'animated_headline' ),
+			]
+		);
+	}
+
 	private function promotion_switcher_control( $title, $id ): string {
 		return '<div class="elementor-control-type-switcher elementor-label-inline e-control-motion-effects-promotion__wrapper">
```

Comment:
Maybe? Could you we replace this `motion-effects` reference with a more general reference?